tests/php/modules/shortcodes/test-class.recipe.php

```diff
@@ -7,17 +7,71 @@
 class WP_Test_Jetpack_Shortcodes_Recipe extends WP_UnitTestCase {
 
 	/**
-	 * After a test method runs, reset any state in WordPress the test method might have changed.
+	 * The tested instance.
+	 *
+	 * @var Jetpack_Recipes
+	 */
+	public $instance;
+
+	/**
+	 * Set up each test.
+	 *
+	 * @inheritDoc
 	 */
 	public function setUp() {
-		// Run hook to load shortcode.
-		do_action( 'init' );
+		parent::setUp();
+		$this->instance = new Jetpack_Recipes();
+		$this->instance->action_init();
+	}
 
-		// Reset data.
-		wp_reset_postdata();
+	/**
+	 * Tear down after each test.
+	 *
+	 * @inheritDoc
+	 */
+	public function tearDown() {
+		wp_dequeue_script( 'jetpack-recipes-js' );
+		wp_dequeue_script( 'jetpack-recipes-printthis' );
```

Comment:
It's a little strange that these are needed, but the tests fail locally without them.